index.js

```diff
@@ -7,25 +7,24 @@ const {
     registerUser,
     updateUser,
     logout,
-    checkIfLoggedIn,
+    isUserLoggedIn,
 } = require('./businessRules/users');
 const { getUsers } = require('./utils/storage');
-const { deleteUser } = require('./models/users');
+const { deleteUser, getUser } = require('./models/users');
 const {
-    ideasDonationSum,
-    ideasSumDifference,
-    createDonation,
+    getTotalSumDonatedForIdea,
+    createDonationByUnregisteredUser,
+    createDonationByRegisteredUser,
 } = require('./businessRules/donations');
 const bodyParser = require('body-parser');
-const { updateIdea, deleteIdea } = require('./models/ideas');
+const { updateIdea, deleteIdea, getAllIdeas } = require('./models/ideas');
```

Comment:
`getAllIdeas` can be renamed to `getIdeas`

Reply:
Renamed